src/test/regress/expected/isolation_merge_1.out

```diff
@@ -0,0 +1,5 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-begin s1-upd-ins s2-result s1-commit s2-result
+setup failed: ERROR:  MERGE is not supported on PG version: 13
```

Comment:
I'd probably avoid the explicit PG version in the output so that we can avoid adding `src/test/regress/expected/isolation_merge_1.out` and `src/test/regress/expected/isolation_merge_replicated_1.out`?